lib/passports.js

```diff
@@ -101,14 +101,26 @@ module.exports = {
                 passport.tokens = JSON.stringify(query.tokens)
               }
 
-              // Save any updates to the Passport before moving on
-              app.services.FootprintService.update('Passport', passport.id, passport) .then(passport => {
-                app.services.FootprintService.find('user', passport[0].user.id)
-                  .then(user => {
-                    next(null, user)
-                  })
-                  .catch(next)
-              }).catch(next)
+             // TODO: Must be tested for check if ok for both Sequelize and Waterline
+             // Save any updates to the Passport before moving on
+              app.services.FootprintService.find('Passport', passport.id)
```

Comment:
Sorry, I created the pull request and committed other 2 changes on master
Only a04e06e & 78a6eec are related to this pull request
The other 2 changes is a WIP related to Sequelize (actually this trailpack doesn't work on Sequelize due to some FootprintService differences compared to Waterline)
Refuses without problems this PR, in these days I will create the different branches with both the fix and create other 2 PRs